### PR TITLE
RHIROS-701 | Add latest RHEL releases for filtering

### PR DESCRIPTION
**Pull request by saltgen**
Changes:
1. Updated OS_VERSIONS with RHEL 8.6, 9.0

Additional:
- NA

### CODE REVIEW

File: ros/api/v1/hosts.py [hunk 1]
```diff
@@ -41,7 +41,9 @@
     '8.2': {"name": "RHEL", "major": 8, "minor": 2},
     '8.3': {"name": "RHEL", "major": 8, "minor": 3},
     '8.4': {"name": "RHEL", "major": 8, "minor": 4},
-    '8.5': {"name": "RHEL", "major": 8, "minor": 5}
+    '8.5': {"name": "RHEL", "major": 8, "minor": 5},
+    '8.6': {"name": "RHEL", "major": 8, "minor": 6},
+    '9.0': {"name": "RHEL", "major": 8, "minor": 0},
```
Comment (upadhyeammit) [inline ros/api/v1/hosts.py:46]:
```suggestion
    '9.0': {"name": "RHEL", "major": 9, "minor": 0},
```

**Review by upadhyeammit — CHANGES_REQUESTED** [review]

File: ros/api/v1/hosts.py (hunk 1 above)
Comment (saltgen) [reply ros/api/v1/hosts.py:46]: @upadhyeammit Oops! Thanks Amit

Comment (upadhyeammit) [reply ros/api/v1/hosts.py]: I think we don't need the `,` for last row? and I missed this in suggestion :| 
~~~
'9.0': {"name": "RHEL", "major": 9, "minor": 0}
~~~

Comment (saltgen) [reply ros/api/v1/hosts.py]: I prefer to add trailing commas such that the next addition is easier and also avoids a syntax error.

Reference: https://peps.python.org/pep-0008/#when-to-use-trailing-commas

Open to suggestions :slightly_smiling_face:

Comment (upadhyeammit) [reply ros/api/v1/hosts.py]: then it makes sense!

**Review by upadhyeammit — APPROVED** [review]

**Review by r14chandra — CHANGES_REQUESTED** [review]
We need to add the versions in openapi file as well - [openapi.json#L828](https://github.com/RedHatInsights/ros-backend/blob/main/ros/openapi/openapi.json#L828)

**Comment by saltgen** [comment]
> We need to add the versions in openapi file as well - [openapi.json#L828](https://github.com/RedHatInsights/ros-backend/blob/main/ros/openapi/openapi.json#L828)

Thanks @r14chandra!

**Review by r14chandra — APPROVED** [review]
Looks good

**Comment by ShivamPandya** [comment]
Hello,
As we have tested on RHEL8.6 and RHEL 9.0, we can move forward with that

**Comment by kgaikwad** [comment]
@saltgen  - I would like to know what else is pending for this pull-request or can we go ahead with merge?
Only thing I noticed that 9.1 is available [link](https://access.redhat.com/products/red-hat-enterprise-linux/) so do you think we can include this option as well?

File: ros/api/v1/hosts.py [hunk 2]
```diff
@@ -41,7 +41,9 @@
     '8.2': {"name": "RHEL", "major": 8, "minor": 2},
     '8.3': {"name": "RHEL", "major": 8, "minor": 3},
     '8.4': {"name": "RHEL", "major": 8, "minor": 4},
-    '8.5': {"name": "RHEL", "major": 8, "minor": 5}
+    '8.5': {"name": "RHEL", "major": 8, "minor": 5},
+    '8.6': {"name": "RHEL", "major": 8, "minor": 6},
+    '9.0': {"name": "RHEL", "major": 9, "minor": 0},
```
Comment (kgaikwad) [inline ros/api/v1/hosts.py:46]:
```suggestion
    '9.0': {"name": "RHEL", "major": 9, "minor": 0},
    '9.1': {"name": "RHEL", "major": 9, "minor": 1},
```

File: ros/openapi/openapi.json [hunk 3]
```diff
@@ -825,7 +825,9 @@
                             "RHEL 8.2",
                             "RHEL 8.3",
                             "RHEL 8.4",
-                            "RHEL 8.5"
+                            "RHEL 8.5",
+                            "RHEL 8.6",
+                            "RHEL 9.0"
```
Comment (kgaikwad) [inline ros/openapi/openapi.json:830]:
```suggestion
                            "RHEL 9.0",
                            "RHEL 9.1"
```

**Review by kgaikwad — CHANGES_REQUESTED** [review]

**Comment by saltgen** [comment]
> @saltgen - I would like to know what else is pending for this pull-request or can we go ahead with merge? Only thing I noticed that 9.1 is available [link](https://access.redhat.com/products/red-hat-enterprise-linux/) so do you think we can include this option as well?

@kgaikwad I have left a question for @ShivamPandya regarding the same on Slack, I will share the link for the same with you.

File: ros/openapi/openapi.json (hunk 3 above)
Comment (saltgen) [reply ros/openapi/openapi.json:830]: As discussed on Slack, proceeding w/o this change for now.

**Pull request merged**